### ISSUE TITLE
Tab Key Doesn't Work on Form Within GLightbox Inline Content Modals

### ISSUE THREAD

**Issue by fatihturan** [biati-digital/glightbox#88]
Hi,

I'm using the Glighbox feature for opening form modal with inline content feature but I can't use the Tab key to navigating on inputs.

**Comment by biati-digital** [comment]
This should be fixed in the latest version 2.0.6, please note that an input must be focused in order to work otherwise the tab key will focus the Lightbox prev, next and close button.

**Comment by onetrev** [comment]
And just as a tip for someone looking to do this (set focus on the input in the modal), you need to do something like this:

```
lightbox.on('open', () => {
  function setFocus() {
    var element = document.querySelector('.ginlined-content .search-input');
    element.focus();
  }
  setFocus()
});
```
